example-apps/swirlds-platform-test-gui/src/main/java/module-info.java

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+module com.swirlds.platform.test.gui {
+    requires transitive com.swirlds.common;
+    requires transitive com.swirlds.platform.core;
+    requires transitive com.swirlds.state.api;
```

Comment:
```suggestion
    requires com.swirlds.common;
    requires com.swirlds.platform.core;
    requires com.swirlds.state.api;
```

I _think_ that's how it should be as this is an "application" and there is no "api" scope on the Gradle size. Unfortunately, you do not get a warning/error for this (https://github.com/gradlex-org/java-module-dependencies/issues/46).

Reply:
Ended up needing to remove this example app either way.